pyembree/trianges.pyx

```diff
@@ -16,7 +16,7 @@ def run_triangles():
     
 cdef unsigned int addCube(rtcs.RTCScene scene_i):
     cdef unsigned int mesh = rtcg.rtcNewTriangleMesh(scene_i,
-                rtcg.RTCGEOMETRY_STATIC, 12, 8, 1)
+                rtcg.RTC_GEOMETRY_STATIC, 12, 8, 1)
```

Comment:
Great catch!

Reply:
Thank you. I think you can rename the file `trianges.pyx` in `triangles.pyx`.

BTW, the example `attenaute.py` also creates an error

    >python attenuate.py
    Ran region 1 in 0.075 s
    Ran region 2 in 0.039 s
    Ran region 3 in 0.005 s
    Traceback (most recent call last):
      File "attenuate.py", line 60, in <module>
        maxdist[exists] = triangles[intersects[gi],:,0]  # get x coord
    ValueError: shape mismatch: value array of shape (1565,3) could not be broadcast to indexing result of shape (1565,)